### ISSUE TITLE
Loss converges slow, and the training period seems too long

### ISSUE THREAD

**Issue by Inosonnia** [Duankaiwen/CenterNet#64]
I run the demo code (CenterNet-104) with batch_size = 6, single GPU with 24G Mem.
I notice that the loss converges slowly and changes around 15, then goes down to around 10 after training for half a day, like this:

  0%|                                   | 47/480000 [06:08<950:02:11,  7.13s/it]
training loss at iteration 50: 16.207754135131836                               
focal loss at iteration 50:    15.354342460632324                               
pull loss at iteration 50:     0.12169381231069565                              
push loss at iteration 50:     0.4843452572822571                               
regr loss at iteration 50:     0.2473718523979187 

BTW, the predicted training time seems too long, any suggestions?

Thank you!

**Comment by Maxxxyu** [comment]
The training period is very long and tried to use multiple gpus does not help. I tried to use 8 rtx 2080ti but the training time is even longer than using single gpu. However, increase the batch size can help increase the speed.
